CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 - Removed method Get_alert_log(), method was no longer functional
 
+### Changed
+
+- updated uri paths to use updated `/api/v#`
```

Comment:
To give customers who might see this a clearer picture of why this change, I'd say: `updated URI paths to use new standardized versioning scheme`

Reply:
agreed, updated